origins_workflow/tests/src/Nightwatch/Tests/moderationSidebar/01.newToDraft.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  '@tags': ['origins', 'origins_workflow', 'debug'],
+
+  before: function (browser) {
+    // Resize default window size.
+    browser.resizeWindow(1600, 2048);
+  },
+
+  'CREATE NEW DRAFT': browser => {
+    // Login as an editor.
+    browser
+      .drupalLogin({ name: process.env.NW_TEST_USER_PREFIX + '_editor', password: process.env.TEST_PASS })
+      .drupalRelativeURL('/node/add/article');
+
+    // Load page objects.
+    const pageObjects = browser.page.ModerationSidebar();
+
+    pageObjects.createArticleNode('draft');
```

Comment:
This is the kind of structure I wanted to see more of in the tests: as much webdriver API detail abstracted away into the page object file. This makes the test more readable, re-usable and maintainable. A shame it was such a headache to work in alongside the others.